ethereum/executionclient/src/integration-test/java/tech/pegasys/teku/ethereum/executionclient/rest/RestBuilderClientTest.java

```diff
@@ -177,7 +177,7 @@ void registerValidators_success() {
         .satisfies(
             response -> {
               assertThat(response.isSuccess()).isTrue();
-              assertThat(response.getPayload()).isNull();
```

Comment:
i think i prefer getter naming...

Reply:
I don't think there is value in customizing getter methods for Record classes. By default they will have the field name so in this case I'd keep it as `payload()`.